src/components/Dashboard/CrossingDetailPage/CrossingDetails.js

```diff
@@ -265,7 +265,7 @@ class CrossingDetails extends Component {
               />
             )}
             <div className="CrossingDetails__communities">
-              <Label>Communities</Label>
+              <Label>{!addMode ? "Communities" : "Community"}</Label>
```

Comment:
Just want to confirm that we only allow adding to one community when adding a crossing.

Reply:
For now, yes. That's always been true, but previously the UI attempted to add additional communities anyway, which just threw errors.